packages/plugin-core/src/views/utils.ts

```diff
@@ -100,7 +101,7 @@ export class WebViewUtils {
     key,
     webviewView,
   }: {
-    ext: DendronExtension;
+    ext: IDendronExtension;
```

Comment:
The new lookup view also calls this method. It currently grabs the instance of the extension with the deprecated bare `getExtension` method. Would changing the signature here cause any issue there?

Reply:
If it's compiling, it should be fine - changing this siganture shouldn't change anything functionally.

Reply:
can confirm it works fine